src/index.js

```diff
@@ -17,7 +17,6 @@ addBtn.addEventListener('click', (e) => {
       index: id,
     };
     listArray.push(task);
-    /* loadData(); */ // call to create the task element.
 
     // Load from local sttorage
     const tasks = localStorage.getItem('data')
```

Comment:
for the localstorage, I think it will be a good idea to make a function and call it whenever necessary, because you are using the same code on multiple of lines and it shows your code is not DRY.

Also for the naming best practices while naming a boolean variable  (e.g. notDeletet) it will be better to name something like (isDeleted).

Reply:
For the `Local Storage` observation, I really don't know how to do it, because every time I call it, storage is in a different `const` for making different tasks every time... Before this observation y try to think of this solution and unfortunately I get stuck.

For the name of the variable, I try to put some meaning, because in that array I'm going to save all elements there are not erased by the user. If I use "isDeleted", maybe is going to be contra-intuitive.